app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateCreditMemoPartialRefundVirtualProductCustomOptionCustomStock.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminCreateCreditMemoPartialRefundVirtualProductCustomOptionCustomStock" extends="AdminCreateCreditMemoTotalRefundVirtualProductCustomOptionDefaultStock">
+        <annotations>
+            <stories value="Virtual Product Custom Stock."/>
+            <title value="Create Partial Refund Credit Memo with Virtual Product with Custom Option on Custom stock."/>
+            <description value="Verify, admin able to create credit memo with partial refund with Virtual product with custom option on custom stock."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2427051"/>
```

Comment:
This is already linked to `AdminCreateCreditMemoPartialRefundVirtualProductCustomStock`

Please investigate and update links

Reply:
I've checked on 2.3-developer branch, and links are different:
AdminCreateCreditMemoPartialRefundVirtualProductCustomStock - has this link https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/1695157
and mentioned above test with custom option has this link - https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2427051